submissions/shramkoweb/memory-game/main.js

```diff
@@ -0,0 +1,109 @@
+const HIDE_TIMEOUT = 500;
+const RESET_TIMEOUT = 700;
+const SHOW_ALL_CARDS_TIMEOUT = 1000;
+const CARDS_COUNT_PER_FLIP = 2;
+
+const contentElement = document.querySelector('.content');
+const cellElements = contentElement.querySelectorAll('.cell');
+const resultElement = document.querySelector('.result');
+
+const state = {
+    isBoardBlocked: false,
+    firstCard: null,
+    secondCard: null,
+    cardsLeft: cellElements.length,
+};
+
+const resetBoardState = () => {
+    state.secondCard = null;
+    state.firstCard = null;
+    state.isBoardBlocked = false;
+};
+
+const hideAllCards = () => {
+    state.isBoardBlocked = false;
+    cellElements.forEach(card => {
+        card.classList.remove('opened');
+    });
+};
+
+const getFlexOrder = (cardsCount) => String(Math.floor(Math.random() * cardsCount));
```

Comment:
If you use grid style, can you rename the function to `getGridOrder`

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/order
Yeah rename to `getOrder` - because order use in flex & grid =)